oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -27,10 +27,10 @@ Context
 *******
 edX/2U has a volunteer Security Working Group (SWG) that has historically handled the triage process for any open
 source vulnerabilities. This process included digesting emails that were sent to security@edx.org, assessing the
-severity of the reported vulnerability, and then triaging the disclosure to the owners/maintainers of the affected
+severity of the reported vulnerability, and then training the disclosure to the owners/maintainers of the affected
```

Comment:
This should not be changed. Triaging is a word, it's a form of https://www.merriam-webster.com/dictionary/triage

Reply:
Oops, I misread it.

Reply:
fixed.